tests/docker/Dockerfile

```diff
@@ -63,7 +63,7 @@ LABEL ducker.creator=$ducker_creator
 RUN apt update && apt install -y sudo git netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python3-pip python3-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev iperf traceroute iproute2 iputils-ping && apt-get -y clean
 RUN python3 -m pip install -U pip==21.1.1;
 # NOTE: ducktape 0.11.4 requires python3.9
-RUN pip3 install --upgrade cffi virtualenv pyasn1 boto3 pycrypto pywinrm ipaddress enum34 debugpy && pip3 install --upgrade "ducktape==0.11.4"
+RUN pip3 install --upgrade cffi virtualenv pyasn1 boto3 pycrypto pywinrm ipaddress enum34 debugpy psutil && pip3 install --upgrade "ducktape==0.11.4"
```

Comment:
oh, trunk does not have such issue since it fixes the `ducktype` version. Maybe we should backport KAFKA-17520 to 3.9, 3.8 and 3.7. WDYT?

Reply:
Ok, I will close this PR and file a new PR to branch  3.9, 3.8 and 3.7.